docs/examples/rag_weaviate.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Ag9kcX2B_atc"
+      },
+      "source": [
+        "# Performing RAG over PDFs with Weaviate and Docling\n",
+        "## A recipe 🧑‍🍳 🐥 💚\n",
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/your-username/your-repo/blob/main/notebook.ipynb)\n",
```

Comment:
Colab URL should be the targeted URL from Docling's `main` branch so that it works once merged.

Reply:
Changed Colab URL to: https://colab.research.google.com/github/DS4SD/docling/blob/tree/main/docs/examples/rag_weaviate.ipynb

Reply:
@m-newhauser I cannot seem to find the updates you are referring to — could it be they were not pushed successfully?

Reply:
Sorry @vagenas, forgot to push them! Can you see them now? https://github.com/m-newhauser/docling/tree/add-weaviate-docling-rag-recipe

Reply:
Yes, thanks!